src/index.js

```diff
@@ -105,8 +105,6 @@ const handleAddTask = (name = '', completed = false, newTask = true) => {
   }
 };
 
-// this is were the java script code sarts to execute
-
 input.addEventListener('input', (e) => {
   newTodo = e.target.value;
 });
```

Comment:
- [ ] It is impossible to use in-line comments because there are no corresponding changes commented on to do it, please 🙏🏽 comment on all the changes when you modify your code for good practices. 
- [ ] Please add a new JavaScript file imported as a module, it is a project requirement 👇🏽 We could check about modules on this [link](https://javascript.info/modules-intro).

> Add a new JavaScript file and import it as a module

- [ ] Please use `Array.filter()` method for the "Cear all completed" button, it helps you to delete the completed tasks because now it is deleting everything.

![image](https://user-images.githubusercontent.com/58642949/215906308-5aaf3773-8589-479d-81f1-6b4da24bef67.png)